### PR TITLE
Add bio for Lucie Molkova

### LINKED ISSUE #20: Add a bio file for luciemolkova
- [x] branch
- [x] bio
- [x] pull req
- [x] feedback
- [x] merge

### PR DESCRIPTION
**Pull request by luckamolkova**
This pull request creates bio for me

Closes #20

### CODE REVIEW

File: luciemolkova.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Lucie Molkova
+
+* Name: Lucie Molkova
```
Comment (mun1r0b0t) [inline luciemolkova.md:3]: @luciemolkova  just comment on mine... :)

Comment (mun1r0b0t) [inline luciemolkova.md:3]: @luciemolkova  what's your favorite desert, not dessert?

Comment (luckamolkova) [inline luciemolkova.md:3]: @munircontractor Ok, ok, added now. You happy?

Comment (mun1r0b0t) [inline luciemolkova.md:3]: @luciemolkova  +1

**Pull request merged**